README.md

```diff
@@ -1,83 +1,80 @@
-Scel
-====
+# scel - sclang-mode for emacs
 
 SuperCollider/Emacs interface
 
+## Installation
 
-Installation requirements
--------------------------
+The repository contains two subprojects. `/sc` contains the SuperCollider code
+required to implement the emacs interface. `/el` contains the emacs-lisp
+implementation of the mode. Emacs and SuperCollider each have their own package
+managers, so it is required to install each half separately.
 
-For the HTML help system, you will need emacs-w3m support.
+### Installing scel quark
 
+The `scel` Quark is required for emacs to communicate with sclang. Evaluate this in the SuperCollider GUI:
 
-Installation (default)
-----------------------
-
-By default emacs-lisp files are installed in
-
-`$prefix/share/emacs/site-lisp`
-
-SuperCollider files are put in
+``` supercollider
+Quarks.install("https://github.com/supercollider/scel");
+```
 
-`$prefix/share/SuperCollider/Extensions/scide_scel`
+### Installing the emacs mode
```

Comment:
Writing from an emacs newbie perspective (well, I forgot everything I've done with it years ago), it might be beneficial to indicate that these go into the init/preferences file, instead of being invoked once.